modin/core/storage_formats/pandas/parsers.py

```diff
@@ -663,43 +663,66 @@ class ParquetFileToRead(NamedTuple):
 
 @doc(_doc_pandas_parser_class, data_type="PARQUET data")
 class PandasParquetParser(PandasParser):
+    @staticmethod
+    def read_row_group_chunk(
```

Comment:
Should this be made a private method?

Reply:
Yeah it's an implementation detail so it should be.